src/templates/base/base.html

```diff
@@ -611,6 +622,17 @@
             }
           }
         });
+
+        document.querySelectorAll('[data-event]').forEach(trackableElement => {
+          trackableElement.addEventListener("click", function (event) {
```

Comment:
To confirm, this won't double-fire for `<option>`s because those are change events and not clicks?

Also one nit about consistent use of single quotes:
```suggestion
          trackableElement.addEventListener('click', function (event) {
```

Reply:
Fixed.